tests/src/test/scala/whisk/core/admin/WskAdminTests.scala

```diff
@@ -91,4 +91,44 @@ class WskAdminTests
         val ns = wsk.namespace.whois()
         wskadmin.cli(Seq("user", "get", ns)).stdout.trim should be(wskprops.authKey)
     }
+
+    it should "block and unblock a user respectively" in {
+        val wskadmin = new RunWskAdminCmd {}
+        val auth = AuthKey()
+        val subject = Subject().asString
+        val other = Subject().asString
+        val commonNamespace = "testspace"
+        try {
+            wskadmin.cli(Seq("user", "create", subject, "-ns", commonNamespace, "-u", auth.compact))
```

Comment:
i would rename `subject` and `other` with `user1` and `user2` (or `subject1` and `subject2`) :)